en/docs/references/ballerina-expression-syntax.md

```diff
@@ -71,237 +71,237 @@ The boolean data type has one of the two possible values, `true` or `false`.
   false
   ```
 
-## Accessing Variables
+## Accessing variables
 
 ### Using the variable value
 
 Variables keep values in memory for later access. In an expression syntax-supported input field, you can refer to these variables directly using the variable name.
 
 #### Example
 
-- Converting the value of the integer variable (`total`) to a string value.
+- This converts the value of an integer variable (`total`) to a string:
```

Comment:
In other places, we have newline before the example. Shouldn't we add new line here too?

Reply:
Yes, we need. 
Added a new line.